rust/parquet_derive/README.md

```diff
@@ -0,0 +1,98 @@
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# Parquet Derive
+
+A crate for deriving `RecordWriter` for arbitrary, _simple_ structs. This does not generate writers for arbitrarily nested
+structures. It only works for primitives and a few generic structures and
+various levels of reference. Please see features checklist for what is currently
+supported.
+
+Derive also has some support for the chrono time library. You must must enable the `chrono` feature to get this support.
+
+## Usage
+Add this to your Cargo.toml:
+```toml
+[dependencies]
+parquet = "0.13.0"
+parquet_derive = "0.13.0"
+```
```

Comment:
Ah, we also need to update https://github.com/apache/arrow/blob/master/dev/release/00-prepare.sh#L122-L140 .

We can use the following code instead of the code:

```shell
sed -i.bak -E \
  -e "s/^([^ ]+) = \".+\"/\\1 = \"${version}\"/g" \
  -e "s,docs\.rs/crate/([^/]+)/[^)]+,docs.rs/crate/\\1/${version},g" \
  */README.md
rm -f */README.md.bak
git add */README.md
```

Reply:
OK, pushed up!